src/main/scala/nl.knaw.dans.easy.managedeposit/EasyManageDepositApp.scala

```diff
@@ -98,11 +99,15 @@ class EasyManageDepositApp(configuration: Configuration) extends DebugEnhancedLo
     "End of error report."
   }
 
-  def cleanDepositor(depositor: Option[DepositorId], age: Int, state: String, onlyData: Boolean): Try[String] = {
-    for {
-      _ <- deleteDepositFromDepositsDir(sword2DepositsDir, depositor, age, state, onlyData)
-      _ <- deleteDepositFromDepositsDir(ingestFlowInbox, depositor, age, state, onlyData)
-    } yield "Execution of clean: success "
+  def cleanDepositor(depositor: Option[DepositorId], age: Int, state: String, onlyData: Boolean, doUpdate: Boolean, newStateLabel: ScallopOption[String], newStateDescription: ScallopOption[String], output: Boolean): Try[String] = Try {
```

Comment:
Don't use any of the Scallop details in this part of the code. That is all wrapped up in the `CommandLineOptions` class and converted in `Command`. Here you should use `Option` instead.

```suggestion
  def cleanDepositor(depositor: Option[DepositorId], age: Int, state: String, onlyData: Boolean, doUpdate: Boolean, newStateLabel: Option[String], newStateDescription: Option[String], output: Boolean): Try[String] = Try {
```

Reply:
done